src/animation/animation.cpp

```diff
@@ -56,21 +58,18 @@ Animation::~Animation()
 }
 
 
-static const float ANIMATION_FPS = 30.0f;
-
-
 void Animation::getPose(float time, Pose& pose, Model& model) const
 {
 	PROFILE_FUNCTION();
 	if(model.isReady())
 	{
-		int frame = (int)(time * ANIMATION_FPS);
+		int frame = (int)(time * m_fps);
```

Comment:
does frame have to be an integer? you do multiple time conversion int <-> float. at least in this function you could do only one:  float frame_count = float(m_frame_count);